src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -680,6 +680,7 @@ private ResumptionTokenType buildResumptionTokenFromRequest(Request request, Str
     extraParams.put(OFFSET_PARAM, String.valueOf(cursor + returnedCount));
     extraParams.put(REQUEST_ID_PARAM, requestId);
     extraParams.put(NEXT_RECORD_ID_PARAM, nextInstanceId);
+    extraParams.put(EXPIRATION_DATE_RESUMPTION_TOKEN, String.valueOf(Instant.now().plusSeconds(RESUMPTION_TOKEN_TIMEOUT)));
```

Comment:
If you will take a look at the variable names you will notice that they have a similar naming pattern. 
Please, rename it to EXPIRATION_DATE_PARAM.